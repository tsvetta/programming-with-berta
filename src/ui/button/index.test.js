import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import Button from 'UI/button';

describe('Button', () => {
  it('should render', () => {
    const component = shallow(
      <Button type='button'>Я кнопочка</Button>
    );

    expect(shallowToJson(component)).toMatchSnapshot();
  });
});
