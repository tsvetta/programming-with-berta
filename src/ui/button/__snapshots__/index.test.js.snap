// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should render 1`] = `
<button
  className="button"
  type="button"
>
  Я кнопочка
</button>
`;
